properties.yaml

```diff
@@ -20,4 +40,11 @@ characters:
       name: Yvain
       file: yvain.json
 audio:
-  folder: sounds
\ No newline at end of file
+  folder: sounds

```

Comment:
from my understanding that PR also removed any sound related code, is that still relevant ?